ground-control/sql/queries/configs.sql

```diff
@@ -0,0 +1,32 @@
+-- name: CreateConfig :one
+INSERT INTO groundcontrol.configs (config_name, registry_url, config, created_at, updated_at)
```

Comment:
```suggestion
INSERT INTO configs (config_name, registry_url, config, created_at, updated_at)
```

no need for  groundcontrol  

update  all queries accordingly

Reply:
I just did that since codacy was complaining about best practice. I can revert if it's not required.

Reply:
I can't find any docs regarding it. it could be hallucinating